src/PointwiseFunctions/AnalyticData/SelfForce/GeneralRelativity/CircularOrbitCoeffs.cpp

```diff
@@ -0,0 +1,646 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "PointwiseFunctions/AnalyticData/SelfForce/GeneralRelativity/CircularOrbitCoeffs.hpp"
+
+// NOLINTBEGIN
+namespace GrSelfForce::detail {
+
+int getAreal(int /*m*/, double a, double /*om*/, double r, double th,
```

Comment:
return `void` since the return value, as far as I can tell doesn't mean anything. Yes it's generated code, but we can still massage it a bit to make it easier to work with, I think :)